src/lib.rs

```diff
@@ -246,14 +276,22 @@ impl Default for RngComponent {
 }
 
 /// A [`Plugin`] for initialising a [`GlobalRng`] into a Bevy `App`.
+#[derive(Debug)]
 pub struct RngPlugin(Option<u64>);
 
 impl RngPlugin {
-    /// Create a new [`RngPlugin`] instance with an optional seed value.
-    /// Uses a randomised seed if `None` is provided.
+    /// Create a new [`RngPlugin`] instance with a seed value.
     #[inline]
-    pub fn new(seed: Option<u64>) -> Self {
-        Self(seed)
+    #[must_use]
+    pub const fn new(seed: u64) -> Self {
+        Self(Some(seed))
+    }
+
+    /// Create a new [`RngPlugin`] instance with a randomized seed.
+    #[inline]
+    #[must_use]
+    pub const fn empty() -> Self {
```

Comment:
I think this should have the same name as the other similar methods. The internal details of the method doesn't matter to the user of the library, only what its effect. So this causes the `RngPlugin` to initialise into a randomized state, therefore I would call it randomized like the other ones.